.gitignore

```diff
@@ -0,0 +1,4 @@
+FullWidthTabs/bower_components/
+QuotesRotator/bower_components/
+ViewModeSwitch/bower_components/
+dependencies
```

Comment:
The `.gitignore` file has a format for comments, and you're right to put these definitions in the shared file for the repository. It's much more efficient. You can be even _more_ efficient, though, when you realize that these entries are _path patterns_ that are matched against... See [`.gitignore`](http://git-scm.com/docs/gitignore)